src/traces/parcoords/shaders/context_vertex.glsl

```diff
@@ -1,5 +1,7 @@
 precision highp float;
 
+#pragma glslify: axisY = require("./y.glsl", mats=mats)
```

Comment:
What does the `mats=mats` do here?

Reply:
According to https://github.com/glslify/glslify/blob/master/README.md#passing-references-between-modules that used `to guarantee that two/more different modules are talking about the same reference`.
Obviously now that we have only one vertex shader program there is no need to keep y axis code outside and require it using `glslify`.